config/schema/og.schema.yml

```diff
@@ -24,7 +24,7 @@ og.settings:
     groups:
       type: mapping
       mapping:
-        bundles:
+        node:
```

Comment:
Yeah I think we just need expand the schema. I'll work on that in the
morning.
On 10 Nov 2015 17:45, "Roy Segall" <notifications@github.com> wrote:

> In config/schema/og.schema.yml
> <https://github.com/amitaibu/og/pull/24#discussion_r44438843>:
>
> > @@ -24,7 +24,7 @@ og.settings:
> >      groups:
> >        type: mapping
> >        mapping:
> > -        bundles:
> > +        node:
>
> Tried a couple of things. Maybe you have something in your sleeve.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/24/files#r44438843>.
>